src/groupByKey.ts

```diff
@@ -2,6 +2,17 @@ type GroupsMap<T> = {
   [key: string]: T[];
 };
 
-export function groupByKey(items, key) {
-  // write code here;
+export function groupByKey<T>(items: T[], key: keyof T): GroupsMap<T> {
+  return items.reduce((obj: GroupsMap<T>, item: T) => {
+    const acc = obj;
```

Comment:
Don't cut variable names. What does it mean `acc`? 
Try to use more meaningful variable naming